src/lib.rs

```diff
@@ -1,133 +0,0 @@
-//! `nu_formatter` is a library for formatting nu.
-//!
-//! It does not do anything more than that, which makes it so fast.
-use config::Config;
-use formatting::{add_newline_at_end_of_file, format_inner};
-use log::{debug, trace};
-use std::fs::File;
-use std::io::Write;
-use std::path::PathBuf;
-
-pub mod config;
-mod formatting;
-
-/// format a Nushell file inplace
-pub fn format_single_file(file: &PathBuf, config: &Config) {
-    let contents = std::fs::read(file)
-        .unwrap_or_else(|_| panic!("something went wrong reading the file {}", file.display()));
-
-    let formatted_bytes = add_newline_at_end_of_file(format_inner(&contents, config));
-
-    if formatted_bytes == contents {
-        debug!("File is already formatted correctly.");
-    }
-
-    let mut writer = File::create(file).unwrap();
-    let file_bytes = formatted_bytes.as_slice();
-    writer
-        .write_all(file_bytes)
-        .expect("something went wrong writing");
-    trace!("written");
-}
-
-/// format a string of Nushell code
-pub fn format_string(input_string: &String, config: &Config) -> String {
-    let contents = input_string.as_bytes();
-    let formatted_bytes = format_inner(contents, config);
-    String::from_utf8(formatted_bytes).unwrap()
-}
-
-#[cfg(test)]
-mod test {
```

Comment:
i do not see these tests anywhere now...

can you put them back in the `nufmt` crate? :innocent:

Reply:
I don't know why tests arent running, I will investigate

Reply:
well, if the tests are not in the source base anywhere, they won't run, right :wink: